backend/README.md

```diff
@@ -0,0 +1,55 @@
+## Run locally
+
+You'll need access to the Firebase project at https://console.firebase.google.com/project/c19-backend.
+
+### Sign into Firebase
+
+From the `backend` directory:
+
+```
+npm install -g firebase-tools
+firebase login
+```
+
+### Start the functions
+
+Install `nvm` if you don't have it already: https://github.com/nvm-sh/nvm#installing-and-updating
+
+Run `nvm use`. If you get an error, you may need to install the configured node version using the `nvm` command in the
+error.
+
+
+```
+cd functions
+yarn install
+firebase functions:config:get > .runtimeconfig.json
+yarn serve
```

Comment:
Boom, I seem to be up and running now! 🎉 

```sh
$ yarn serve
yarn run v1.19.1
$ firebase emulators:start --only functions
i  emulators: Starting emulators: functions
✔  hub: emulator hub started at http://localhost:4400
✔  functions: Using node@8 from host.
✔  functions: functions emulator started at http://localhost:5001
i  functions: Watching "/Users/sprad/code/career/sse/recidiviz/covid19-dashboard/backend/functions" for Cloud Functions...
⚠  Non-existent functions.config() value requested!
   - Path: "service_account.projectId"
   - Learn more at https://firebase.google.com/docs/functions/local-emulator
⚠  Non-existent functions.config() value requested!
   - Path: "service_account.privateKey"
   - Learn more at https://firebase.google.com/docs/functions/local-emulator
⚠  Non-existent functions.config() value requested!
   - Path: "service_account.clientEmail"
   - Learn more at https://firebase.google.com/docs/functions/local-emulator
✔  functions[getFirebaseToken]: http function initialized (http://localhost:5001/c19-backend/us-central1/getFirebaseToken).
✔  emulators: All emulators started, it is now safe to connect.
```

Reply:
> @itsjon - I've got everything up and running locally. Thanks for providing the helpful README! Longer-term, would you envision that this "backend" project would be broken out into its own repo?

Thank you for the review! Yeah, a separate repo was my first thought, but after discussion, we decided to put it in the same one since that probably expedites development in the near-term.

Reply:
Yeah I pushed to keep this in a single repository for now on two counts:
1. If you are working on a new feature that involves both frontend and backend changes, you’d now be working in two separate repos at the same time, managing two separate PRs at the same time, managing keeping those changes in sync at the same time. It seems simpler/faster to stick with a single repo, which is important for a project that relies to some degree on speed.
1. We’ll need to duplicate some of the Auth0 configuration and code across both repositories, presumably

But once things calm down, I'm not categorically opposed to a separate repo!

Reply:
I think we can move the frontend into a subdirectory, so that the directory structue is:

```
frontend/
backend/
shared/ (???)
```

Monorepos are the way to go these days for projects of all sizes -- I'd advocate for this instead of splitting it up!